tests/gatsby-starter-default/package.json

```diff
@@ -6,7 +6,8 @@
   "dependencies": {
     "gatsby": "^1.9.127",
     "gatsby-link": "^1.6.30",
-    "gatsby-plugin-react-helmet": "^1.0.8"
+    "gatsby-plugin-react-helmet": "^1.0.8",
+    "gatsby-source-plone": "git+https://github.com/collective/gatsby-source-plone.git#add_source_dependency"
```

Comment:
@sneridagh won't that always install gatsby-source-plone from master branch? I was thinking about installing the actual code with "npm install <local-package>"

Reply:
I don't get the point. Isn't good to install from master? So it will test the actual status of the plugin.

Reply:
I assume that you are talking about the "npm link". The thing is that you have to build the plugin. Are you going to build the one inside node_modules? No, you won't. The idea is not to push the compiled version to the repo each time you commit.

Reply:
Ok. The thing that I was missing was the "yarn link" call here.